### PR TITLE
Add support for Hades Revisited.

### PR DESCRIPTION
**Pull request by sfence**
Add support for Hades Revisited subgame.

### CODE REVIEW

**Comment by random-geek** [comment]
Thanks, I'll try to look at this soon.

File: morelights/init.lua [hunk 1]
```diff
@@ -115,9 +123,31 @@ if morelights.game == "mineclone2" then
...
+    a.grass = "hades_grass:grass_1"
+    a.cotton = "hades_farming:cotton"
+    a.stick = "hades_core:stick"
+end
+
 -- Use basic_materials brass if available, otherwise register our own.
 if minetest.get_modpath("basic_materials") then
     a.brass = "basic_materials:brass_ingot"
+elseif minetest.get_modpath("basic_materials") then
+    a.brass = "hades_extramaterials:brass_ingot"
```
Comment (random-geek) [inline morelights/init.lua:150]: Is this a mistake? There doesn't appear to be any mod called `hades_extramaterials`.

File: morelights/init.lua [hunk 2]
```diff
@@ -115,9 +123,31 @@ if morelights.game == "mineclone2" then
     a.stick = "mcl_core:stick"
 end
 
+if morelights.game == "hades_revisited" then
+    a.glass = "hades_core:glass"
+    a.glass_pane = "hades_xpanes:pane_flat"
```
Comment (random-geek) [inline morelights/init.lua:128]: HR doesn't seem to have xpanes, should this be regular glass?

**Review by random-geek — COMMENTED** [review]
This is a good start. I'll make some changes myself and then merge this.

File: morelights/init.lua (hunk 2 above)
Comment (sfence) [reply morelights/init.lua:128]: hades_xpanes is part of [Mods pack for Hades Revisited](https://content.minetest.net/packages/SFENCE/hades_extensionmods/).

Comment (random-geek) [reply morelights/init.lua:128]: Oh, I see. I could use the panes from your modpack if available, but I don't want to make `hades_xpanes` a hard dependency.

**Pull request merged**